### PR TITLE
Update to 1.0.27, add fedc annotations, update to GNOME 42 runtime

### LINKED ISSUE #2: Sandbox issue - access home directory
The Flatpack is not allowed to access the home directory of the executing user, so it's not possible to:

1. Save Genius configuration under ~/.gnome2/genius (as the sandbox gets cleaned after termination)
2. Load/Save Files to the home directory.

### PR DESCRIPTION
**Pull request by wjt**
- Config storage fix for #2 from #6 
- Canonicalise manifest & appdata formatting
- Add flatpak-external-data-checker annotations
- Update to 1.0.27
- Update to GNOME 42 runtime

Fixes #2.

### CODE REVIEW

**Comment by wjt** [comment]
OK, the build finished while I was writing that.

Screenshot from 2022-07-13 14-38-43

Still draft-y because the `.gnome2` persist seems deeply dodgy

**Comment by wjt** [comment]
Something is deeply wrong with the plot view though

Screenshot from 2022-07-13 16-23-44

File: org.gnome.Genius.json [hunk 1]
```diff
@@ -11,13 +11,18 @@
         "--socket=x11",
         "--socket=wayland",
         "--own-name=org.gnome.genius",
-        "--persist=.gnome2"
```
Comment (TingPing) [inline org.gnome.Genius.json:14]: Probably leave both to migrate configs?

Comment (wjt) [reply org.gnome.Genius.json:14]: Done

**Comment by TingPing** [comment]
Seems fine to me. Up to you if wayland should be disabled for now, upstream should certainly be notified of that issue.

**Comment by wjt** [comment]
Works perfectly on Wayland today, including moving it between my hidpi and lowdpi monitors. Really strange.

**Comment by wjt** [comment]
The Git repo is actually [on GNOME GitLab](https://gitlab.gnome.org/GNOME/genius) but is not mentioned anywhere on the Genius website.

**Pull request merged**